Tests/CarthageKitTests/XcodeSpec.swift

```diff
@@ -316,6 +316,7 @@ class XcodeSpec: QuickSpec {
 			}
 
 			expect(isExpectedError) == true
+			expect(result.error?.description) == "Dependency \"Swell-0.5.0\" has no shared framework schemes for any of the platforms: Mac"
```

Comment:
Was this change intended for this PR?

Reply:
Yes. This is indented to test the `description` -> `rawValue` change to Platform

Reply:
Ah, okay. Well, divorced from the potential of these `XcodeSpec` changes being removed, I'm fine enough with the changes to [the `zip(with: //` thing](https://github.com/Carthage/Carthage/pull/2374#discussion_r174268071).